docs/book/getting-started/installation/m1-mac-installation.md

```diff
@@ -205,6 +205,19 @@ pip install zenml
 
 And that's it, you have successfully installed ZenML in Python 3.9 under Rosetta.
 
+## Known Limitations
+
+### TensorFlow Integration
```

Comment:
Do you want to mention the special Tensorflow M1 package that exists here? Or say somewhere that we're working to remove this dependency blockage. I.e. explain what options (+ tradeoffs) exist for someone who uses tensorflow but also wants to use ZenML + they have an M1/2 mac.

Reply:
TBH I don't know about this myself so not sure what to write here 😅

Reply:
I added a mention of the two MacOS TF packages and that we are working on resolving this limitation.

Reply:
I sent you something on Discord that maybe you can also link out to somewhere.